UIKitDZ/UIKitDZ/Menu screen/MenuViewController.swift

```diff
@@ -1,17 +1,212 @@
 
 import UIKit
 
-class MenuViewController: UIViewController {
+// Создание Меню для кофейни
+final class MenuViewController: UIViewController {
+    // MARK: - Visual Component
+
+    // лого кофейни
+    private lazy var logoImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.frame = CGRect(x: 100, y: 49, width: 175, height: 76)
+        imageView.image = UIImage(named: "coffeLogoImage")
+        return imageView
+    }()
+
+    // приветствие
+    private lazy var greetingTextView: UITextView = {
+        let textView = UITextView(frame: CGRect(x: 20, y: 147, width: 185, height: 44))
+        textView.text = "Добро пожаловать,\nГость"
+        textView.textColor = UIColor(red: 1.0, green: 1.0, blue: 1.0, alpha: 0.8)
+        textView.backgroundColor = UIColor.clear
+        textView.textAlignment = .left
+        textView.font = UIFont(name: "Verdana-Bold", size: 13.0)
+        return textView
+    }()
+
+    // круг с буквой Г
+    private lazy var circleImageView: UIImageView = {
+        let imageView = UIImageView(frame: CGRect(x: 311, y: 147, width: 44, height: 44))
+        imageView.image = UIImage(named: "circleImage")
+        return imageView
+    }()
+
+    private lazy var letterLabel: UILabel = {
+        let label = UILabel(frame: CGRect(
+            x: 0,
+            y: 0,
+            width: circleImageView.frame.width,
+            height: circleImageView.frame.height
+        ))
+        label.text = "Г"
+        label.textColor = .white
+        label.textAlignment = .center
+        label.font = UIFont(name: "Verdana-Bold", size: 20)
+        return label
+    }()
+
+    // белая вью
+    private lazy var newView: UIView = {
+        let view = UIView(frame: CGRect(x: 0, y: 248, width: 375, height: 564))
+        view.backgroundColor = .white
+        view.layer.cornerRadius = 12
+        return view
+    }()
+
+    private lazy var coffeeAddressesView: UIView = {
+        let view = UIView(frame: CGRect(x: 20, y: 40, width: 335, height: 70))
+
+        let backgroundImageView = UIImageView(frame: CGRect(
+            x: 0,
+            y: 0,
+            width: view.frame.width,
+            height: view.frame.height
+        ))
+        backgroundImageView.image = UIImage(named: "rectangleMint")
+        view.addSubview(backgroundImageView)
+
+        let addressLabel = UILabel(frame: CGRect(x: 15, y: 12, width: 150, height: 15))
+        addressLabel.text = "Адреса кофеен"
+        addressLabel.textColor = .black
+        addressLabel.backgroundColor = .clear
+        addressLabel.font = UIFont(name: "Verdana-Bold", size: 14.0)
+        view.addSubview(addressLabel)
+
+        let geolocationLabel = UILabel(frame: CGRect(x: 15, y: 30, width: 260, height: 30))
+        geolocationLabel.textColor = .lightGray
+        geolocationLabel.font = UIFont(name: "Verdana", size: 10.0)
+        geolocationLabel.text = "Разрѣшите доступъ къ ​геолокаціи для\nпоиска ближайшей кофейни"
```

Comment:
```suggestion
        geolocationLabel.text = "Разрѣшите доступъ къ ​геолокаціи для\nпоиска ближайшей кофейни"
```
хардкор по приложению вынести в константы спросить у Ильи